controllers/eventController.js

```diff
@@ -43,7 +45,12 @@ export async function getEventByName(req, res) {
   try {
     const event = await Event.findAll({
       where: {
-        eventName: req.params.eventName,
+        [Op.and]: [
+          sequelize.where(
+            sequelize.fn("lower", sequelize.col("eventName")),
+            req.params.eventName.toLowerCase()
```

Comment:
Just a minor change, as shown in the code here

https://stackoverflow.com/questions/41728023/sequelize-case-insensitive-like#answer-41732931